proto/fav.proto

```diff
@@ -34,6 +33,6 @@ message GetAllFavoriteRequest {
 }
 
 message GetAllFavoriteResponse {
-    repeated string files = 3;
```

Comment:
If you request all favorite files by the userID, then why would you need to return the userID with each file? 
They will all be the same (The userIDs). Right?

Reply:
I changed it. Now it will return only an array of fileID